integration-tests/hardhat.config.ts

```diff
@@ -13,13 +13,13 @@ const config: HardhatUserConfig = {
   },
   networks: {
     optimism: {
-      url: 'http://localhost:8545',
+      url: process.env.L2_URL || 'http://localhost:8545',
       ovm: true,
     },
   },
   solidity: '0.7.6',
   ovm: {
-    solcVersion: '0.7.6',
+    solcVersion: '0.7.6-allow_kall_2', // temporary until we fix the build for 0.7.6
```

Comment:
Should this be an env var with this as the default value?

Reply:
No this is not supposed to be configurable. Kall support is exposed by the compiler now so this change is no longer required.